examples/example-full-configuration.yaml

```diff
@@ -23,7 +23,10 @@ global:
   saml_admin_login_role: "Shibboleth-Administrator"
   # base Taupage AMI to search for
   base_ami:
-    name: "Taupage-AMI-*"
+    default_channel: "Taupage-AMI-*"
+    channels:
```

Comment:
why did you need the channels in sevenseconds? If one account need a other taupage app then you can set the base_ami for this account

```
foobar:
  base_ami:
    name: "TaupageStaging-AMI-*"
```

Reply:
To correctly setup Taupage for AWS accounts, we either share the image during build time (for existing accounts), or use sevenseconds on new accounts which deals with it.
Unfortunately, the logic it uses doesn't really scale. It looks for one existing image (`Taupage-AMI-…`) and only if that isn't present shares every image. Moreover, there's no explicit step for this, it's just a side effect of bastion host/VPC initalization. This means that 1) it's impossible to automatically share staging AMIs and 2) if there's any issues the account might be stuck in a weird state.
I've changed the logic to be a lot more straightforward: we define explicit channels in the configuration, fetch the latest AMIs for each channel and then share the AMIs with the accounts if they are not shared. The latest AMI fetch is also done once to avoid bumping into rate limits, since I don't think overriding the Taupage configuration for an account is something that we ever needed, and there's already some parts of sevenseconds that are done globally.